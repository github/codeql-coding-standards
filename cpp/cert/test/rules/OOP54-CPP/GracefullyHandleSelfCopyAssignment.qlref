rules/OOP54-CPP/GracefullyHandleSelfCopyAssignment.ql