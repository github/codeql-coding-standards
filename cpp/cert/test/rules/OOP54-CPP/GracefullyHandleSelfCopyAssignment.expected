| test.cpp:57:6:57:14 | operator= | User defined copy or user defined move does not handle self-assignment correctly.  |
| test.cpp:66:6:66:14 | operator= | User defined copy or user defined move does not handle self-assignment correctly.  |
